test/helpers.html

```diff
@@ -205,6 +205,22 @@
     return visibleRows;
   };
 
+  window.isWithinParentConstraints = (el, parent) => {
+    ['top', 'bottom', 'left', 'right'].forEach(constraint => {
+      if (el.getBoundingClientRect[constraint] !== parent.getBoundingClientRect[constraint]) {
+        return false;
+      }
+    });
+
+    return true;
+  };
```

Comment:
This function will always return `true` is the `return false` statement is within the `forEach` callback. Maybe using `every` or `some` would make more sense here